a10_neutron_lbaas/etc/config.py

```diff
@@ -166,5 +166,31 @@
 #     'vthunder_tenant_password': '',
 
 #     'vthunder_management_network': 'private-mgmt',
-#     'vthunder_data_networks': [ 'vip-net', 'member-net' ]
+#     'vthunder_data_networks': [ 'vip-net', 'member-net' ],
+
+# # Explicitly set gateway on the management network
+#     'vthunder_management_gateway': '192.168.100.1',
+
+# # License the launched instances
```

Comment:
It won't reject those comments being indented to the same level as the fields. Which is the style in the block above.